packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/Payment/index.tsx

```diff
@@ -18,6 +18,7 @@ import { Props } from '../template.success'
 const RightArrowIcon = styled(Icon)<{
   disabled?: boolean
 }>`
+  transform: rotate(180deg);
```

Comment:
I tried all the different "right" arrows we had, but none of them were the larger skinnier ones so I just used the "arrow-back" icon and rotated it to get the desired style.